### PR TITLE
[Validation] Move EcalDigis sources in to plugins dir

### PR DESCRIPTION
**Pull request by smuzaffar**
`Validation/EcalDigis` generates a plugin so this PR proposes to move all src/headers files in to plugins directory.

### CODE REVIEW

**Comment by smuzaffar** [comment]
please test

**Comment by jfernan2** [comment]
+1

File: Validation/EcalDigis/plugins/EcalDigisValidation.h [hunk 1]
```diff
@@ -37,7 +37,7 @@
 #include "DataFormats/EcalDigi/interface/EcalDigiCollections.h"
 #include "SimDataFormats/CrossingFrame/interface/CrossingFrame.h"
 #include "SimDataFormats/CrossingFrame/interface/MixCollection.h"
-#include <Validation/EcalDigis/interface/EcalDigisValidation.h>
+#include "EcalDigisValidation.h"
```
Comment (perrotta) [inline Validation/EcalDigis/plugins/EcalDigisValidation.h:40]: Is this including itself?

File: Validation/EcalDigis/plugins/EcalEndcapDigisValidation.h [hunk 2]
```diff
@@ -25,7 +25,7 @@
 #include "DataFormats/EcalDigi/interface/EEDataFrame.h"
 #include "DataFormats/EcalDigi/interface/EcalDigiCollections.h"
 
-#include <Validation/EcalDigis/interface/EcalEndcapDigisValidation.h>
+#include "EcalEndcapDigisValidation.h"
```
Comment (perrotta) [inline Validation/EcalDigis/plugins/EcalEndcapDigisValidation.h:28]: Is this including itself?

File: Validation/EcalDigis/plugins/EcalBarrelDigisValidation.h [hunk 3]
```diff
@@ -31,7 +31,7 @@
 #include <map>
 #include "DQMServices/Core/interface/DQMEDAnalyzer.h"
 
-#include <Validation/EcalDigis/interface/EcalBarrelDigisValidation.h>
+#include "EcalBarrelDigisValidation.h"
```
Comment (perrotta) [inline Validation/EcalDigis/plugins/EcalBarrelDigisValidation.h:34]: Is this including itself?

**Comment by perrotta** [comment]
@smuzaffar , just in case you did not notice my comments [above](https://github.com/cms-sw/cmssw/pull/34904#pullrequestreview-732226366)

**Comment by perrotta** [comment]
Thank you @smuzaffar 
However, there are still two more self-inclusions to clean up (see above)

File: Validation/EcalDigis/plugins/EcalDigisValidation.h (hunk 1 above)
Comment (smuzaffar) [reply Validation/EcalDigis/plugins/EcalDigisValidation.h]: yes it was, I have removed it now.

File: Validation/EcalDigis/plugins/EcalEndcapDigisValidation.h (hunk 2 above)
Comment (smuzaffar) [reply Validation/EcalDigis/plugins/EcalEndcapDigisValidation.h]: yes it was, I have removed it now.

**Comment by smuzaffar** [comment]
please test

**Comment by perrotta** [comment]
+1
- @jfernan2 changes since last dqm signature are only the removal of a few duplicate include: I assume that signature can be confirmed

**Comment by perrotta** [comment]
merge

**Comment by jfernan2** [comment]
+1
For the records

**Pull request merged**